### PR TITLE
SHARKS-7600 add filtering out headers for resend

### PR DESCRIPTION
**Pull request by Pefes**

### CODE REVIEW

File: kouncil-backend/src/main/resources/application.yaml [hunk 1]
```diff
@@ -11,3 +11,4 @@ logging:
 kouncil:
   topics:
     exclude-regex-patterns: __.*
+    resend-headers-to-keep: requestId, version
```
Comment (dmejer) [inline kouncil-backend/src/main/resources/application.yaml:14]: This should be provided from outside like bootstrapServers, default no value

File: kouncil-backend/src/main/java/com/consdata/kouncil/topic/TopicService.java [hunk 2]
```diff
@@ -48,6 +52,9 @@ public class TopicService {
     private final MessagesHelper messagesHelper;
     private static final int RESEND_MAX_POLL_RECORDS = 100;
 
+    @Value("${kouncil.topics.resend-headers-to-keep}")
+    private String[] resendHeadersToKeep;
```
Comment (dmejer) [inline kouncil-backend/src/main/java/com/consdata/kouncil/topic/TopicService.java:56]: Maybe one service with all custom configuration? + validation @Validated from spring

File: kouncil-frontend/libs/resend-events/src/lib/resend/resend.component.ts [hunk 3]
```diff
@@ -103,6 +103,10 @@ import {ResendFormService} from './resend-form.service';
           </mat-form-field>
         </div>
 
+        <div>
```
Comment (dmejer) [inline kouncil-frontend/libs/resend-events/src/lib/resend/resend.component.ts:106]: do you need this div?

**Review by dmejer — CHANGES_REQUESTED** [review]

File: kouncil-backend/src/main/java/com/consdata/kouncil/topic/TopicService.java (hunk 2 above)
Comment (Pefes) [reply kouncil-backend/src/main/java/com/consdata/kouncil/topic/TopicService.java:56]: Imo that's too much for now, especially if this is temporary solution and passing empty list of headers to keep makes sense (validation is not useful)

**Review by dmejer — APPROVED** [review]

**Pull request merged**